### PR TITLE
Task #51833

### PR DESCRIPTION
**Pull request by thonght-3051**
## Related Tickets
- [#51833](https://edu-redmine.sun-asterisk.vn/issues/51833)

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

### CODE REVIEW

**Comment by thonght-3051** [comment]
ready

File: app/helpers/admin/orders_helper.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+module Admin::OrdersHelper
+  def display_color_status status
+    case status
+    when "pending"
+      "text-primary"
```
Comment (ghost) [inline app/helpers/admin/orders_helper.rb]:
```suggestion
     {"pending" => "text-primary"}[status]
```

**Review by ghost — COMMENTED** [review]
cmt

**Comment by thonght-3051** [comment]
ready

**Review by ghost — COMMENTED** [review]
chết rubocop rồi em

**Comment by thonght-3051** [comment]
ready

**Review by ghost — COMMENTED** [review]
merged

**Pull request merged**